bin/sandbox

```diff
@@ -99,7 +99,11 @@ unbundled bin/rails generate solidus:install \
   --auto-accept \
   $@
 
+unbundled bundle add lookbook listen actioncable --group development,test
+ruby -e "File.write('config/routes.rb', File.read('config/routes.rb').gsub(/draw do/, %{draw do\n  mount Lookbook::Engine, at: '/lookbook' if Rails.env.development?}))"
```

Comment:
Just a note on this, as we discussed during the live review, this could be moved from here probably 🤔
Even if to be honest, I don't have a better idea for the moment on how to do it differently.

Reply:
I'll try to move both operations to the solidus_admin install generator 👍